specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1470,11 +840,6 @@ components:
           $ref: "#/components/schemas/TypeOppervlak"
         perceelnummerVerschuiving:
           $ref: "#/components/schemas/TypePerceelnummerVerschuiving"
-        nummeraanduidingIdentificaties:
-          type: "array"
-          description: "Identificatie van een adres uit de basisregistratie adressen en gebouwen (BAG). Dit kan worden gebruikt om een URL op bouwen of om in te vullen in de tempated HAL link naar de BAG."
-          items:
-            type: "string"
         stukIdentificaties:
```

Comment:
Stukken zijn verwijderd. Moet dit dan ook niet worden verwijderd?

Reply:
Klopt. Thanks, is nu aangepast.